cmd/minishift/cmd/preflight_artifacts.go

```diff
@@ -28,6 +28,10 @@ import (
 
 // preflightChecksForArtifacts is executed once artifacts are cached.
 func preflightChecksForArtifacts() {
+	// If --skip-preflight-check flag is set to true means do not check
+	if viper.GetBool(configCmd.SkipPreflightChecks.Name) {
```

Comment:
Create a function:

```
function shouldPreflightChecksBeSkipped() {
   return viper.GetBool(configCmd.SkipPreflightChecks.Name)
}
```

and change this code to:
```
   if  shouldPreflightChecksBeSkipped() {
      return
   }
```

Reply:
@piyush1594 adding comments almost always means the code is cryptic (not clear in intent), so that needs to be solved. the first step is always to prevent magic values; we have done this with the `configCmd.SkipPreflightrChecks.Name`. But sinc ethis code is used several times, we have to find a way to avoid duplication... and the best way is to refactor into a small function. This prevents the need to look for logic if something ever changes... but usage is also easier to track by a simple grep/find on the file. Now, this can also be tested in exclusion or potentially be mocked. Besides, is it in this piece of code really necessary to know `--skip-preflight-check` is used to enable the skip? And to be honest, that was already changed...

https://github.com/minishift/minishift/pull/2325/files#diff-73778a5d257cfccc3347694f8c39f2bcR125

into `--skip-startup-checks`. Right? Comments seldom get as much the attention as they need or that code gets.